homework/6. classes/task_06_01.py

```diff
@@ -0,0 +1,118 @@
+import random
+
+
+class Matrix:
+    def __init__(self, rows, cols=None):
+        """
+        Инициализация объекта Matrix.
+        :param rows: int или список списков int
+        :param cols: int
+        """
+        if isinstance(rows, int):  # Если rows - это целое число
+            self.rows = rows  # Устанавливаем количество строк
+            self.cols = cols  # Устанавливаем количество столбцов
+            # Создаем матрицу с случайными числами от 1 до 10
+            self.matrix = [
+                [random.randint(1, 10) for _ in range(cols)] for _ in range(rows)]

```

Comment:
Перенос обусловлен стилем написания кода или нет? Просто выглядит странно.
Нагляднее было бы в одну строчку
```suggestion
                [[random.randint(1, 10) for _ in range(cols)] for _ in range(rows)]

```
Либо с явно выделенным открытием и закрытием списка
```suggestion
                [
                    [random.randint(1, 10) for _ in range(cols)] for _ in range(rows)
                ]

```
Но это на мой вкус

Reply:
\+
не проверила за автоматическим форматером кода по pip8